css/src/markdown.scss

```diff
@@ -1,6 +1,11 @@
 /**
  * Markdown rendering
  */
+
+$color-lightgrey: #bbbbbb;
+$color-main-background: #ddd;
```

Comment:
See comments, but I don't think line is needed since it's defined here:

https://github.com/nextcloud/server/blob/master/core/css/css-variables.scss